_episodes/12-cluster.md

```diff
@@ -158,242 +168,11 @@ $ free -g
-
-> ## All mixed up
->
-> Lola needs to obtain a file called `results.data` from a remote machine that is called 
-> `safe-store-1`. This machine is hidden behind the login node `{{ site.login_host }}`. 
-> However she mixed up the commands somehow that are needed to get the file onto her laptop. 
-> Help her and rearrange the following commands into the right order!
->
-> ~~~
-> $ ssh lola@`{{ site.login_host }}`
-> $ logout
-> $ scp lola@`{{ site.login_host }}`:results.data .
-> $ scp lola@safe-store-1:results.data .
-> ~~~
-> {: .language-bash}
->
-> > ## Solution
-> > ~~~
-> > $ ssh lola@`{{ site.login_host }}`
-> > $ scp lola@safe-store-1:results.data .
-> > $ logout
-> > $ scp lola@`{{ site.login_host }}`:results.data .
-> > ~~~
-> > {: .language-bash}
-> {: .solution}
-{: .challenge}
-
-
-> ## Who is hanging around ?
->
-> The `w` utility displays a list logged-in users and what they are currently doing. Use it to check:
->
-> 1. that nobody but yourself is logged into your laptop/desktop
-> 2. that a lot of people use the login node of your cluster `{{ site.login_host }}`
-{: .challenge}
-
-> ## Where did they go ?
->
-> Rob has a zip file stored under `/tmp/passwords.zip` on the login node of the cluster 
-> `{{ site.login_host }}`. He wants to unzip it on his laptop under `/important/passwords`. 
-> How does he do that?
->
-> 
-> 1.
-> ~~~
-> $ ssh rob@{{ site.login_host }}
-> $ unzip /tmp/passwords.zip
-> ~~~
-> {: .language-bash}
-> 
-> 2.
-> ~~~
-> $ scp {{ site.login_host }}@rob:/tmp/passwords.zip .
-> $ unzip passwords.zip
-> ~~~
-> {: .language-bash}
-> 
-> 3.
-> ~~~
-> $ cd /important/passwords
-> $ scp rob@{{ site.login_host }}:passwords.zip .
-> $ unzip passwords.zip
-> ~~~
-> {: .language-bash}
-> 
-> 4.
-> ~~~
-> $ cd /important/passwords
-> $ scp rob@{{ site.login_host }}:/tmp/passwords.zip .
-> $ unzip passwords.zip
-> ~~~
-> {: .language-bash}
-> 
-> > ## Solution
-> > 
-> > 1. No: Rob only unpacks the zip file, but does not transfer the unpacked files onto his laptop
-> > 2. No: Rob mixed up the syntax for `scp`
-> > 3. No: Rob did not specify the correct path of `/tmp/passwords.zip` on the login node of the 
-> cluster `{{ site.login_host }}`
-> > 4. Yes: you may also use `unzip foo.zip -d /somewhere` if you want to omit the first command
-> {: .solution}
-{: .challenge}
-
 > ## Differences Between Nodes
-> Many HPC clusters have a variety of nodes optimized for particular workloads. Some nodes may have
-> larger amount of memory, or specialized resources such as Graphical Processing Units.
+> Many HPC clusters have a variety of nodes optimized for particular workloads. Some nodes may have larger amount of memory, or specialized resources such as Graphical Processing Units (GPUs).
 {: .callout}
 
-
+With all of this in mind, we will now cover how to talk to the cluster's scheduler, and use it to
+start running our scripts and programs!
 
```

Comment:
There are exercises missing here. One idea: 
> Compare your laptop's number of processors and memory with the numbers you see on the login node! Share your findings with the others.

Reply:
Addressed in https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/8e2ca7059afd9b0080cab714be51e85c712790cf